tests/e2e/multi_svc_test.go

```diff
@@ -0,0 +1,618 @@
+				dumpSvcNsEventsOnTestFailure(clients[i], namespaces[i])
+				framework.Logf("log-after-each-2-4")
+			}
+		*/
+		framework.Logf("log-after-each-3")
+	})
+
+	/* TESTCASE-1
+
+	Create multiple SVCs and create workloads on it
+
+	Steps:
+		1. Verify the SVC account is different for each cluster
+		2. Create sts on each of the clusters and verify that it is successful
+		3. Delete all the sts and pvcs which belong to them
+	*/
+
+	ginkgo.It("[csi-multi-svc] Workload creation on each of the clusters", func() {
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
+		if supervisorCluster {
+			ginkgo.By("CNS_TEST: Running for WCP multiSVC setup")
+		}
+		for n := 0; n < numberOfSvc; n++ {
+			storageClassNames[n] = defaultNginxStorageClassName
+			profileID := e2eVSphere.GetSpbmPolicyID(storagePolicyNames[n])
+			scParametersList[n][scParamStoragePolicyID] = profileID
+			// create resource quota
+			createResourceQuota(clients[n], namespaces[n], rqLimit, storageClassNames[n])
+			c1 := clients[n]
+			ns1 := namespaces[n]
+			scSpec := getVSphereStorageClassSpec(storageClassNames[n], scParametersList[n], nil, "", "", false)
+			sc, err := clients[n].StorageV1().StorageClasses().Create(ctx, scSpec, metav1.CreateOptions{})
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			defer func() {
+				err := c1.StorageV1().StorageClasses().Delete(ctx, sc.Name, *metav1.NewDeleteOptions(0))
+				gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			}()
+
+			ginkgo.By("Creating service")
+			service := CreateService(namespaces[n], clients[n])
+			defer func() {
+				deleteService(ns1, c1, service)
+			}()
+
+			statefulset := GetStatefulSetFromManifest(namespaces[n])
+			ginkgo.By("Updating replicas to 3 and podManagement Policy as Parallel")
+			*(statefulset.Spec.Replicas) = 3
+			statefulset.Spec.PodManagementPolicy = apps.ParallelPodManagement
+			statefulset.Spec.VolumeClaimTemplates[len(statefulset.Spec.VolumeClaimTemplates)-1].
+				Spec.StorageClassName = &storageClassNames[n]
+			statefulset.Spec.VolumeClaimTemplates[len(statefulset.Spec.VolumeClaimTemplates)-1].
+				Spec.StorageClassName = &storageClassNames[n]
+			CreateStatefulSet(namespaces[n], statefulset, clients[n])
+			replicas := *(statefulset.Spec.Replicas)
+			// Waiting for pods status to be Ready
+			fss.WaitForStatusReadyReplicas(clients[n], statefulset, replicas)
+			if n == 1 {
+				os.Setenv(kubeconfigEnvVar, kubeconfig1)
+				framework.TestContext.KubeConfig = kubeconfig1
+				clients[n], err = createKubernetesClientFromConfig(kubeconfig1)
+				gomega.Expect(err).NotTo(gomega.HaveOccurred(),
+					fmt.Sprintf("Error creating k8s client with %v: %v", kubeconfig1, err))
+				// framework.AfterReadingAllFlags(&framework.TestContext)
+			}
+			// err = CheckMountForStsPods(clients[n], statefulset, mountPath)
+			// gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+			ssPodsBeforeScaleDown := fss.GetPodList(clients[n], statefulset)
```

Comment:
tests/e2e/multi_vc_utils.go has lot of sts related utils, much of this code is already there

Reply:
re-used utils method